crates/mesh_drawing/Cargo.toml

```diff
@@ -6,9 +6,10 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bevy = { version = "0.11", default-features = false, features = [
+bevy = { version = "0.12", default-features = false, features = [
     "bevy_render",
+    "bevy_pbr",
```

Comment:
Yep, for `StandardMaterial`

Reply:
Oh right 👍. Weirdly the example was compiling. Also `rust-analyzer` dint seem to catch any errors. Although running `cargo check` fails without `bevy_pbr`. This might because of the full `bevy` version in `dev-dependencies`. Might be an isolation bug. Nvm!